lib/network_interface/worker.ex

```diff
@@ -103,11 +164,21 @@ defmodule Nerves.NetworkInterface.Worker do
     end)
     {:noreply, state}
   end
+
   def handle_info({_, {:exit_status, _}}, state) do
     {:stop, :unexpected_exit, state}
   end
 
+  @typedoc false
+  @type port_resp :: term | no_return
```

Comment:
Just a thought, but `any` seems more intention revealing than `term` to me. They mean the same thing, but I wonder if `any` would be more clear. Thoughts?

Reply:
I think `any` is a good option. Because we really do want to handle `any`thing. :+1: